.github/workflows/cffconvert.yml

```diff
@@ -0,0 +1,19 @@
+name: cffconvert
+
+on:
+  push:
+    paths:
+      - CITATION.cff
```

Comment:
```suggestion
  push:
    paths:
      - CITATION.cff
  pull_request:    
    paths:
      - CITATION.cff
```

Reply:
No, after a bit of thinking I realized your solution is much better. I hadn't noticed the path specification, which is new to me. I'll merge it as proposed :-)